cypress/fixtures/allFarmersGrainResults.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
Thanks for showing me how fixtures operate in Cypress! I feel much more comfortable with them now.